scep/api/api.go

```diff
@@ -359,6 +359,7 @@ func writeResponse(w http.ResponseWriter, res Response) {
 	}
 
 	w.Header().Set("Content-Type", contentHeader(res))
+	w.Header().Set("Cache-Control", "private, no-store")
```

Comment:
@hslatman Would the SCEP response data ever contain sensitive information? From what I can tell it's either a certificate or a list of capabilities and would never contain the challenge secret.

Reply:
Any secrets are encrypted before being sent to the CA, and not returned to clients, so no obvious sensitive information is leaked. There's some bits of SCEP transaction related information that can be extracted, such as transaction ID and nonces, but I don't think these leak information; at least not directly.  

While some responses to GET requests _could_ be cached for a specific amount of time (e.g. `GetCACert`, `GetCACaps`), I feel responses to POST requests must not be cached. The last (informational) SCEP RFC doesn't say much about caching, only that it has caused issues in the past with large requests and when requests weren't actually idempotent. 

We currently support and signal the more modern `POSTPKIOperation` (via POST), so clients generally don't send the large GET requests. The responses for POST requests contain client specific results and are unlikely to be fully idempotent, so IMO shouldn't be cached.

In terms of interoperability: I would have to take a look at other SCEP servers if they ever set cache headers. I think clients should simply ignore them when they don't understand them or can't do anything with them, but SCEP being legacy, there's a lot of old systems that may be doing weird things.

Reply:
@kippmorris7 Let's remove this one and then the PR will be good to merge.

Reply:
I've removed it. To double check my understanding, why did you decide against keeping the headers on the POST responses?